api/src/main/java/org/apache/cloudstack/api/response/SystemVmResponse.java

```diff
@@ -38,14 +38,6 @@ public class SystemVmResponse extends BaseResponseWithAnnotations {
     @Param(description = "the system VM type")
     private String systemVmType;
 
-    @SerializedName("jobid")
-    @Param(description = "the job ID associated with the system VM. This is only displayed if the router listed is part of a currently running asynchronous job.")
-    private String jobId;
```

Comment:
BaseResponseWithAnnotations extends BaseResponse. And BaseResponse already has jobId & jobStatus.